Content.Server/Anomaly/AnomalySystem.Scanner.cs

```diff
@@ -187,7 +187,14 @@ public FormattedMessage GetScannerMessage(AnomalyScannerComponent component)
         {
             msg.AddMarkupOrThrow(Loc.GetString("anomaly-scanner-point-earned", ("point", anomalyComp.PointsEarned)));
             msg.PushNewline();
-            msg.AddMarkupOrThrow(Loc.GetString("anomaly-scanner-anomalite-expected", ("count", int.Min(anomalyComp.PointsEarned / anomalyComp.PointsPerCrystalUnit, anomalyComp.MaxCrystals))));
+            var pointCost = anomalyComp.PointsPerCrystalUnit;
+            var numCrystals = 0;
+            while (pointCost < anomalyComp.PointsEarned && numCrystals < anomalyComp.MaxCrystals)
+            {
+                pointCost += (int)(pointCost * anomalyComp.PointsPerCrystalMult);
+                numCrystals++;
+            }
+            msg.AddMarkupOrThrow(Loc.GetString("anomaly-scanner-anomalite-expected", ("count", numCrystals)));
```

Comment:
Ideally we'd have this implemented in a single helper function that gets called both here and for actually generating the drops. Doing it this way risks causing the anomaly scanner to report an incorrect amount of anomalite if changes are not correctly duplicated between both spots.